crml/governance/src/lib.rs

```diff
@@ -67,6 +78,14 @@ decl_event! {
 		EnactProposal(ProposalId, bool),
 		/// A proposal was vetoed by the council
 		ProposalVeto(ProposalId),
+		/// A referendum was vetoed by vote
+		ReferendumVeto(ProposalId),
+		/// A referendum has been created and will start at start_time
+		ReferendumCreated(ProposalId),
+		/// A referendum has been approved and is awaiting enactment
+		ReferendumApproved(ProposalId),
+		/// Start of the end Referendum Step
+		EnactingReferendum(),
```

Comment:
nit for consistency with `EnactProposal`
```suggestion
		EnactReferendum(),
```